prettier-maven-plugin-integration-tests/src/test/java/com/hubspot/maven/plugins/prettier/AbstractPrettierMojoTest.java

```diff
@@ -123,8 +118,8 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
       }
     });
 
-    String template = Resources.toString(
-        Resources.getResource("pom-template.xml"),
+    String template = new String(
+        AbstractPrettierMojoTest.class.getResourceAsStream("/pom-template.xml").readAllBytes(),
```

Comment:
Similar here, maybe we can extract a method that converts an `InputStream` to a string and ensures that it's closed

Reply:
I ended up creating a private `readString` method to do this, let me know what you think.